src/v2/Components/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -233,6 +233,10 @@ export const ArtworkFilterContextProvider: React.FC<
     false
   )
 
+  useEffect(() => {
+    setAggregations(aggregations)
+  }, [aggregations])
```

Comment:
I'm wondering, if the test failure `Too long with no output (exceeded 10m0s): context deadline exceeded` is valid, it could have something to do with an unintended consequence of this running on mount?

Reply:
Hmm, ok, i try to fix it, thanks

Reply:
https://github.com/artsy/palette/blob/master/packages/palette/src/utils/useUpdateEffect.ts is a common pattern for dealing with that. (Again, this is just a stab in the dark)

Reply:
I'll try it, thank you!

Reply:
The update effect with skipping mount are not work 😞

Reply:
Another solution work.
@dzucconi @damassi check it please)